packages/lab/src/radio-button/RadioButtonIcon.css

```diff
@@ -1,78 +1,89 @@
-/* Styles applied to root component */
+/* Styles applied to radio button icon */
 .saltRadioButtonIcon {
-  --radioButton-icon-fill: transparent;
-  --radioButton-icon-fill-checked: var(--salt-selectable-borderColor-selected);
-  --radioButton-icon-fill-checked-disabled: var(--salt-selectable-borderColor-selectedDisabled);
-  --radioButton-icon-fill-error: var(--salt-status-error-borderColor);
-  --radioButton-icon-fill-error-disabled: var(--salt-selectable-borderColor-selectedDisabled);
+  --radioButton-strokeColor-error: var(--salt-status-error-borderColor);
+  --radioButton-strokeColor-error-disabled: var(--salt-status-error-borderColor-disabled);
 
-  width: var(--saltRadioButton-icon-svg-size, var(--salt-size-selectable));
-  height: var(--saltRadioButton-icon-svg-size, var(--salt-size-selectable));
+  width: var(--salt-size-selectable);
+  height: var(--salt-size-selectable);
   border-radius: 50%;
   flex: none;
   margin-top: var(--salt-size-border);
 }
 
-/* Styles applied to RadioButtonIcon border */
-.saltRadioButtonIcon-border {
-  fill: var(--saltRadioButton-icon-fill, var(--radioButton-icon-fill));
-  stroke: var(--saltRadioButton-icon-strokeColor, var(--salt-selectable-borderColor));
+/* Styles applied to radio button icon circle */
+.saltRadioButtonIcon-circle {
+  fill: var(--saltRadioButton-fill, var(--salt-container-primary-background));
+  stroke: var(--saltRadioButton-strokeColor, var(--salt-selectable-borderColor));
```

Comment:
@navkaur76 has just confirmed the border tokens should be selectable foreground

Reply:
updated ;)